### ISSUE TITLE
[Bug Report][3.3.15] Unexpected page-scrolling when click a <v-list-item> with <v-checkbox-btn>

### ISSUE THREAD

**Issue by catscarlet** [vuetifyjs/vuetify#18223]
### Environment
**Vuetify Version:** 3.3.15
**Vue Version:** 3.3.4
**Browsers:** Firefox 116.0
**OS:** Linux x86_64

### Steps to reproduce
1. Code like that
2. Scroll to somewhere
3. Click one of the <v-list-item>s, no need to click the <v-checkbox-btn>
4. See what happens
5. Scroll to somewhere else, click another <v-list-item> in the other row.

### Expected Behavior
Nothing should happen. No random scrolling.

### Actual Behavior
Page randomly bump into scrolling.
As I researched, the checkbox in that row, always gets the top.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNq9VdtuozAQ/ZWRX9quQmiadh8QqbrazyhVRWBovDEG2Yamivj3HQwk3FZbdaN984zPnDlj7MPzkf3I82VZIPOYbzDNRWjwMZAAfsxLu6Bl6USZNCGXqNqUTarsHWTmvBXGoNKnnbZAnGOA0kkytQnYtcKI5/iTurxl6mMBTaxfuYzxcANcwjpg/Upvjx9UOMANEdRKE+K+n+2JaeSEKh7kbFZwbUDQWHV9EgqNARuhTjiH0/GM9yaDLWgCEvg6HBPsYKvvQ92D+ebKpvCJuKE8x3Ajms83gek8lI9wPM4LrCrwXQvxtwrcr1B8g9XtpXhuL8VzOaLPM9Gd0LvsfXxpYbOBFd0vwINRITQkLeNXuO5OXHf/lcx3527c7MWMdhjtt9nB2ZoZgH259vHoIopQ67nnASDCLQoCEVhgZDCeg5HcYbs/SSJX4zHZWI3v1pMX359w7Brt3tBg6uZDh7GpTJztkmLyy85R7W7fUn238Vvf7bkwhTpSPG+68TTPlIEjeWYCFSQqS+GKrPuqGRUPdjvGJCwEwZpOcWjC65suAqo1hZLnmBwsFAV6sHpYdLmqWVQ2QQFdhU4FW7BGhpOG+fKXziT9OCwZWZjdoK/odfQBI3l1HLCdMbn2XDeKJZXFKHiplhKNK/PUfSKYqwppeIpOnKVP6+V6eU8nok0/vUSdOls6RY2KSALWKrZtDE/IMP/aqkXaDquHpkWbc+im6brHhNulhiUqRyG9F4XqsyONyvpjjbYmo7VHX9GB0w9qr0dnHdWvxYPnTuS/zWzZaqqXumO1YFYotW4q2MtvBSxnmw==)

### Other comments
Here is a video to show what it looks like: https://youtu.be/ajbAArH0zQ4

**Comment by catscarlet** [comment]
As I tested, <v-switch> also has the same issue as <v-checkbox-btn>.

**Comment by catscarlet** [comment]
I realize it will always focus on the first <input> of <v-list-item>.

Reproduction Link: [Reproduction Link](https://play.vuetifyjs.com/#eNq9Vdtu2zAM/RXOL1m3KG562YPhFh32GXUR+MK0WmXJkGQ3QeB/Hy3brW/Aiq7Ym0iRh4cURd6fvJ9FsalK9AIvtJgXIrZ4G0mAMOOVO9CxYqmSNuYSdadySq1eQCr2WFqL2rzedA7iTQao2F7pm8j7qjHlBf6iKI9KH9fQymbHZYaHM+ASLiNv6Bk845EcR3ZjCwplyOJqqB2QaenEOhvpnFZwY0FQWo3/PhYGI29i9WrHOJVnejdLbE0ZEMHdOE1wiW1/jHmP8ltym5vPyI3pMcutaJ9vZmaKWN7C6bRMsK4h9J1JmGjwPwLxDbbnn4Vz/lk4nwf0fiTqCfOkXqZNCzc3sKX+AjxYHUML0iF+BOviFeviv4KF/lLHLdh9YWyxXdMnTJ8TdWCJXXBz/9l9KVOmKRqz9GkARJygICMyFphazJbMKIlxOCLK2DJbLovSLkUy9iiQIiVKZ6gD2BYHMErwDBJR0sBY8rHHonGxVNFlg6CKyXn2Et9htYvtbkWH9w0EcI24WGaa35wIuxr059lsG77ldD52d+NR2hR0PEudSom3xUAybYZ+d7jb4fII/XazhP5g35BoUs2LNhrPC6UtnGg77KGGvVY5rGhJrdpU8eCuM9zHpSCzNlIW2/jrWS8B+dpSyzeZZnVTc3rA63Wvq9tD7RQkUNP3LLy119JgeVxsfhslaUU6MBrW7oI6M+jhI4/oNXLkPVlbmMD300ySW4aCV3oj0fqyyP07MvN1KS3PkWUqv7vcXG6uqCLGDtUbNDlLqIoGNYFEXsfYhbF8T53w11CdpYuwvW5DdDpGv8c0MWbYPgWsUDON1H4a9XtTmrgN05pczVLrSl9TwWkVP5tJrdNmAgRw35P8t5wdWgP10ESs154jSqFbD+/hD5Keosc=)

**Comment by KaelWD** [comment]
The list focuses the first focusable child when it itself is focused, but this should probably only happen on keyboard focus.

**Comment by catscarlet** [comment]
> The list focuses the first focusable child when it itself is focused, but this should probably only happen on keyboard focus.

Yeah it shouldn't happen when using mouse. Now I'm having difficulty to click **checkbox** when the first `<input>` isn't in screen.

I found a temporary solution that somehow, **setting `<input>`'s tabindex to `-1`** could prevent this issue happen but it would also break the keyboard `tab` navigation, which is the featureI really want to have in my project.

**Comment by DurtyFree** [comment]
Seems like this bug got introduced with Vuetify Release 3.3.2

I can confirm it with our application

**Comment by igal1c0de4n** [comment]
Also bumping against this issue.
Is there expectation for a fix to be shipped in the near future? Maybe a reliable workaround (?)
I'm trying to understand if downgrading to v3.3.1 is my best course of action atp

**Comment by J-Sek** [comment]
Quick workaround to suppress (prevent `onFocus` in [VList:211](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VList/VList.tsx#L211))

```
@click=""
:ripple="false"
```

[demo](https://play.vuetifyjs.com/#eNp1U9tuwjAM/RUvmwRItGhj2wOiiH3DHgGJElyICEmUpIgJ8e/LhUtTtpemPr7k+NiZnYjRdPClVH6okYzI2OJe8dLiZC4AxoesVCr8BoNKYUsmUF+gAHJm7M2+IRlzle4owCGrpC7mhAET8PY5J03naIc/3tdCLbMcHb70xeD55cTOyzRkSjmjOxfSytRMqZBaldxg09mg6she24VnpVGhWCfu2PQW6W4lj9nKitQJQCUPTZmaUjQmJRGI7OUaeXYoee3ZGORILZMiZ4Lyeo2my3qPWdNarR2pVrKVmw3H72uJPzIHaXODZJhXsDGf+xgv6HXUzkxn7ZG4CklVZxrqtLZg0NZxU9heSW3hBBorOEOl5R46brs63umqGh986QEKH9WdLXreWdUioo+dwikyYxV07yIGZZpS3uLgfkeMcje18yrGLeruEYoJHOGpKMApGrLPgG5r/q2Vq9psvfwx2B/uMx5EKZwI5Nwnw3yYv5N4vn6QfljEfvLcIrb4BVwuDgg=)
